### PR TITLE
Create Visual Tests for SegmentedSwitch

### PR DESCRIPTION
**Pull request by domihustinova**
Closes https://kiwicom.atlassian.net/browse/FEPLT-2059

### CODE REVIEW

**Review by DSil — COMMENTED** [review]
LGTM 👍 I am wondering if we should have two more cases, that were relevant in the past:

- error with focus (make sure the focus remains blue)
- selected and hovered (make sure the text remains blue)

WDYT of adding them?

File: packages/orbit-components/src/CarrierLogo/CarrierLogo.ct.tsx-snapshots/visual-CarrierLogo-CarrierLogo-1-Desktop-linux.png
Comment (mvidalgarcia) [inline packages/orbit-components/src/CarrierLogo/CarrierLogo.ct.tsx-snapshots/visual-CarrierLogo-CarrierLogo-1-Desktop-linux.png:1]: I guess this shouldn't be here, not sure why it was generated.

File: packages/orbit-components/src/Modal/Modal.ct.tsx-snapshots/visual-Modal-ModalVisualDefaultStory-extraLarge-1-Small-Mobile-linux.png
Comment (mvidalgarcia) [inline packages/orbit-components/src/Modal/Modal.ct.tsx-snapshots/visual-Modal-ModalVisualDefaultStory-extraLarge-1-Small-Mobile-linux.png:1]: Same here.

**Comment by mvidalgarcia** [comment]
Feel free to squash the last two commits.

File: packages/orbit-components/src/CarrierLogo/CarrierLogo.ct.tsx-snapshots/visual-CarrierLogo-CarrierLogo-1-Desktop-linux.png
Comment (domihustinova) [reply packages/orbit-components/src/CarrierLogo/CarrierLogo.ct.tsx-snapshots/visual-CarrierLogo-CarrierLogo-1-Desktop-linux.png:1]: 🙈 Deleted.

**Comment by mvidalgarcia** [comment]
LGTM, @DSil I'll let you do the final approval as you've taken a look already.

**Review by DSil — APPROVED** [review]
👍 🚀

**Pull request merged**